### DISCUSSION TITLE
Is this even legal? Can you get IP banned?

### DISCUSSION THREAD

**Discussion by ghost** [charlesbel/Microsoft-Rewards-Farmer#402] [General]
I have been questioning whether this is even legal or not. I read a bit of the ToS but to my limited understanding and knowledge of the law, i concluded that it is because botting may count as damaging the company and damaging a company as far as i know is an illegal activity. 

Now if Microsoft bans your IP i assume the answer is yes but to your experience and knowledge:

- How do you reduce the chances or prevent it?
- Do you use a proxy or a VPN?
- Has it happened to you? if so, what have you done and how has it affected you?

I think this is a very interesting and beneficial discussion and i hope it is for you and for this community. Thanks.

**Comment by aliasfoxkde** [comment]
It's not illegal per se, but it is against Microsoft's TOC (as well as most others) to automate or use bots. All they will do is block your account and you will loose the points earned on it (just the one usually). And yes, you can reduce your chances of being caught by simply seeming human (run the bot slower, stager accounts runs, randomize accounts used, etc.). 

Redeeming is the number one way you will get caught, because that's when they are paying out. They have little issue with faking their own numbers, so from a pure use standpoint, MS loves what bots bring to the table.. makes them seem as though their services are being used more often than they are. I digress..

So redeem only as often as you need to (go for larger gift cards), redeem for sweepstakes (I don't even think they do SMS authentication for those), use a VPN (US only), or Proxy. Per their TOC's you are allowed to have 5 accounts tied to the same IP and it's unlikely they will ban your IP itself, just the flagged MS Rewards account.
